extensions/wikia/MercuryApi/models/MercuryApi.class.php

```diff
@@ -146,6 +146,9 @@ public function getWikiVariables() {
 			'theme' => SassUtil::normalizeThemeColors( SassUtil::getOasisSettings() ),
 			'tracking' => [
 				'vertical' => HubService::getVerticalNameForComscore( $wgCityId ),
+				'comscore' => [
+					'c7Value' => AnalyticsProviderComscore::getC7Value(),
```

Comment:
This can return `false`, I wonder if it wouldn't be better to handle in the API instead of the client? I guess it depends on what we want to do in that case.